app/test_split_integer.py

```diff
@@ -2,20 +2,31 @@
 
 
 def test_sum_of_the_parts_should_be_equal_to_value() -> None:
-    pass
+    parts = split_integer(50, 6)
+
```

Comment:
I think no need to create variable 

Write it in one line

Reply:
Ok.